__tests__/models/ItemModel.test.ts

```diff
@@ -0,0 +1,280 @@
+import { ItemModel } from "../../app/src/models/item"; // adjust path as needed
+
+//general items
+import restrictedAndMissingMediaManifest from "../__mocks__/data/collectionsApi/manifests/item/restricted_and_missing_media.json";
+import restrictedManifest from "../__mocks__/data/collectionsApi/manifests/item/restricted.json";
+
+//images
+import singleImageCaptureManifest from "../__mocks__/data/collectionsApi/manifests/item/image/single_capture.json";
+import multiImageCaptureManifest from "../__mocks__/data/collectionsApi/manifests/item/image/multiple_capture.json";
+
+//audio
+import singleAudioCaptureManifest from "../__mocks__/data/collectionsApi/manifests/item/audio/single_capture.json";
+
+//video
+//note: no multi video capture manifest bc we allegedly don't have Items that have multiple video captures
+import singleVideoCaptureManifest from "../__mocks__/data/collectionsApi/manifests/item/video/single_capture.json";
+import missingMediaVideoCaptureManifest from "../__mocks__/data/collectionsApi/manifests/item/video/missing_capture.json";
+
+describe("ItemModel - Image - Single Capture", () => {
+  const uuid = "a9c43f00-c600-012f-59c3-58d385a7bc34";
+
+  let item: ItemModel;
+
+  beforeEach(() => {
+    item = new ItemModel(uuid, singleImageCaptureManifest);
+  });
+
+  describe("Non-Manifest/Metadata related fields", () => {
+    it("sets the correct UUID", () => {
+      expect(item.uuid).toBe(uuid);
+    });
+    it("generates the correct link and manifestURL", () => {
+      const env = process.env.APP_ENV || "";
+      const baseLink =
+        env === "development" || env === "qa"
+          ? `https://qa-digitalcollections.nypl.org/items/${uuid}`
+          : `https://digitalcollections.nypl.org/items/${uuid}`;
```

Comment:
I'm unsure about this. What's the intention of setting the baselink to different domains in the test? Tests should have a dedicated env so it's always consistent, but seeing this, I can imagine that it won't be testing the same thing all the time; it's going to test qa and prod separately based on the APP_ENV value.

Separate, use something like `process.env.TEST_ENV` rather than `APP_ENV` if possible, just to distinguish the run time value for the test. (but this might be moot depending on the feedback from the first comment above)

Reply:
this might be residue from previous, outdated work